SignalServiceKit/src/Messages/Attachments/OWSAttachmentDownloads.m

```diff
@@ -593,8 +615,10 @@ - (void)downloadFromLocation:(NSString *)location
                 return;
             }
 
+            job.progress = progress.fractionCompleted;
```

Comment:
I wonder if it'd be beneficial to just pass back the entire NSProgress rather than a number... but maybe YAGNI, since we're not (yet) doing anything like aggregating the results.